packages/cli/templates/platform-app-tailwind/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "sample-app",
+  "private": true,
+  "author": "Frontify Developers <developers@frontify.com>",
+  "version": "0.0.0",
```

```suggestion
  "version": "1.0.0",
```
We are not the author of newly created block ;P